### ISSUE TITLE
Change class filenames to same case as class name (PSR-4 Precondition)

### ISSUE THREAD

**Issue by ximex** [Admidio/admidio#644]
eg. `tableaccess.php` to `TableAccess.php`

There should be no problems because all classes are loaded with the autoloader function. But we could add a fallback into the autoloader

in autoloader.php
```php
$libFiles = array(
    ADMIDIO_PATH . FOLDER_CLASSES . '/' . $className . '.php',
    ADMIDIO_PATH . FOLDER_LIBS_SERVER . '/jquery-file-upload/server/php/' . $className . '.php',
    ADMIDIO_PATH . FOLDER_LIBS_SERVER . '/monolog/src/' . str_replace('\\', '/', $className) . '.php',
    ADMIDIO_PATH . FOLDER_LIBS_SERVER . '/phpass/' . strtolower($className) . '.php', // old phpass password hashing lib for backward compatibility
    ADMIDIO_PATH . FOLDER_LIBS_SERVER . '/phpmailer/class.' . strtolower($className) . '.php',
    ADMIDIO_PATH . FOLDER_LIBS_SERVER . '/psr/log/' . str_replace('\\', '/', $className) . '.php',
    ADMIDIO_PATH . FOLDER_LIBS_SERVER . '/securimage/' . strtolower($className) . '.php',
    ADMIDIO_PATH . FOLDER_LIBS_SERVER . '/tcpdf/' . strtolower($className) . '.php',
    ADMIDIO_PATH . FOLDER_LIBS_SERVER . '/zxcvbn-php/src/' . substr(str_replace('\\', '/', $className), 9) . '.php',
    ADMIDIO_PATH . FOLDER_CLASSES . '/' . strtolower($className) . '.php' // Backwards compatibility fallback
);
```

**Comment by Fasse** [comment]
We had several problems with camelstyle filenames on servers in the past.

What should be the advantage of this change?

**Comment by ximex** [comment]
Could you show me this problems? couldn't imagine what the problem should be.
If there were problems, we should still/again have them, because in the libs folder are files with upper case chars.

The advantage is that we support the PSR-4 standard.
and my IDE wouldn't cry any longer about this ;-)

**Comment by Fasse** [comment]
Ok, we could change it but keep in mind this in mind if someone has problems with not existing class files.

But please change this after we had merged all branches for version 3.3
